src/orion/core/worker/consumer.py

```diff
@@ -100,6 +100,52 @@ def consume(self, trial):
             trial.status = 'broken'
             self.experiment.update_trial(trial, status=trial.status)
 
+    def get_execution_environment(self, trial, results_file='results.log'):
+        """Set a few environment variables to allow users and
+        underlying processes to know if they are running under orion
+
+        Parameters
+        ----------
+        results_file: str
+           file used to store results, this is only used by the legacy protocol
+
+        trial: Trial
+           reference to the trial object that is going to be run
+
+        Notes
+        ----
+        This functions define the environment variables described below
+
+        ORION_PROJECT: str
+           name of the experiment the user is currently working on.
```

Comment:
Wouldn't the name of the Experiment the user is currently working on be stored inside `ORION_EXPERIMENT` below?